src/app/component/shared/items/item/item.component.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  ViewEncapsulation
+} from "@angular/core";
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
+import { Href } from "src/app/interfaces/menusInterfaces";
+import { StrongRoute } from "src/app/interfaces/strongRoute";
+
+@Component({
+  selector: "app-items-item",
+  template: `
+    <li class="list-group-item clearfix">
+      <fa-icon id="icon" [icon]="icon"></fa-icon>
+      <span id="name">
+        {{ name }}
+      </span>
+      <span id="value" class="badge badge-pill badge-secondary float-right">
+        {{ value }}
+      </span>
+    </li>
+  `,
+  styles: [
+    `
+      li {
+        font-size: 0.925rem;
+      }
+    `
+  ],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ItemComponent {
+  @Input() icon: IconProp;
+  @Input() name: string;
+  @Input() value: string | number;
+  @Input() uri: Href | StrongRoute;
```

Comment:
Uri value is not being used, add a test to determine what type is added and place a href on the item so that clicking will follow the uri.